### PR TITLE
Fix images width in UG

### PR DESCRIPTION
**Pull request by sebastiantoh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -344,11 +344,13 @@ Format: `COMMAND help`
 
 * If `COMMAND` is not present,
     * list the available commands and the link to the User Guide.
-![result for 'help'](images/helpAllMockup.png)
+
```
Comment (AaronnSeah) [inline docs/UserGuide.md]: the lack of new line before the image is intentional. Will it mess with the width attribute?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -344,11 +344,13 @@ Format: `COMMAND help`
...
 * If `COMMAND` is not present,
     * list the available commands and the link to the User Guide.
-![result for 'help'](images/helpAllMockup.png)
+    
+<img src="images/helpAllMockup.png" alt="result for 'help'" width="400px">
 
 * If `COMMAND` is present
     * list the command description and example usage.
-![result for 'help command'](images/helpPerCommandMockup.png)
+
```
Comment (AaronnSeah) [inline docs/UserGuide.md]: the lack of new line before the image is intentional. Will it mess with the width attribute?

File: docs/UserGuide.md (hunk 1 above)
Comment (hakujitsu) [reply docs/UserGuide.md]: Based on what I see, it looks fine when the page is published, and the spacing is the same as the other pictures.

File: docs/UserGuide.md (hunk 2 above)
Comment (hakujitsu) [reply docs/UserGuide.md]: Based on what I see, it looks fine when the page is published, and the spacing is the same as the other pictures.

File: docs/UserGuide.md (hunk 1 above)
Comment (sebastiantoh) [reply docs/UserGuide.md]: Oh, my bad. I didn't know the new line would affect the formatting. Thought it was just code styling. Will remove it!

Comment (AaronnSeah) [reply docs/UserGuide.md]: no problem! just checking 👍

**Review by AaronnSeah — APPROVED** [review]

**Pull request merged**